experiment/run_experiment.py

```diff
@@ -168,31 +168,6 @@ def validate_fuzzer(fuzzer: str):
         raise Exception('Fuzzer "%s" does not exist.' % fuzzer)
 
 
-def validate_fuzzer_config(fuzzer_config):
```

Comment:
If we are no longer validating fuzzer configs maybe we should stop supporting them?

Reply:
yes we dont support them at all, not accepted through command line. we just create fuzzer configs files with fuzzer attribute to pass to dispatcher.

Reply:
Even this I'd like to get rid of if possible then. There's no reason to support it anymore.
This can come later if it would be easier.

Reply:
Yes next CL, i move faster in increments rather than gigantic cls.